docs/cip/README.md

```diff
@@ -2305,50 +2305,46 @@ incentives and conflict resolution trade-offs.
-> "The changes to the ledger rules are implemented and deployed on Cardano
-> mainnet by a majority of the network", or "The following key projects have
-> implemented support for this standard".
-
-- [ ] The revised `cardano-node` implementations pass the node-level conformance
-      test suites.
-- [ ] Audit.
-- [ ] Successful operation in testnet environments.
-- [ ] Community agreement on the settings for the Leios protocol parameters.
+The proposal will be considered active once the following criteria are met:
+
+- [ ] Protocol performance validated through load tests in a representative
+      environment.
+- [ ] Required changes are documented in an node implementation-independent way
+      via the
+      [Cardano blueprint](https://cardano-scaling.github.io/cardano-blueprint/)
+- [ ] Community agreement on initial Leios protocol parameters.
+- [ ] A peer-reviewed implementation of a Leios-enabled node is available.
+- [ ] Successful operation with open participation in testnet environments over
+      several months.
+- [ ] Key infrastructure signalled readiness for Leios-enhanced throughput.
+- [ ] Hard-fork enabling Leios is successfully executed on mainnet.
 
 **Implementation plan**
 
-> [!NOTE]
->
-> Either a plan to meet those criteria or `N/A` if not applicable.
->
-> This sub-section should define the plan by which a proposal will meet its
-> acceptance criteria, when applicable. More, proposals that require
-> implementation work in a specific project may indicate one or more
-> implementors. Implementors must sign off on the plan and be referenced in the
-> document's preamble.
->
-> In particular, an implementation that requires a hard-fork should explicitly
-> mention it in its _'Implementation Plan'_.
-
-- [ ] Detailed node-level (as opposed to this protocol-level) specification.
-- [ ] Develop node-level conformance test suite.
-- Consider developing a "quick and dirty" implementation for large scale
-  experiments.
-- Coordinate with related activities on other protocol enhancements.
-  - Compatibility between Peras, Leios, and Genesis.
-  - Common design and implementation for certificates, voting, and related key
-    registration: Mithril, Peras, Leios, and partner chains.
-  - https://github.com/berewt/CIPs/blob/master/CIP-xxxx/CIP-xxxx.md is relevant
-- Triage by intersect Core Infrastructure, Consensus, Ledger, and Network
-  functions.
+Key steps on the roadmap to realize Leios, somewhat ordered but not sequential,
+are:
+
+- [x] Simulations to confirm general feasibility using a model of mainnet.
+- [x] Ecosystem impact analysis and establish a threat model.
+- [ ] Coordinate with related activities on other protocol enhancements
+      (Phalanx, Peras, Mithril).
+- [ ] Detailed node-level (as opposed to this protocol-level) technical
+      specification.
+- [ ] Complete ΔQ analysis of new/changed network interactions.
+- [ ] Complete consensus-level formal protocol specification in Agda.
```

Comment:
What about network and ledger specs?

Reply:
I agree about the ledger part (I kind of implied it).
Did we ever "drill down" to the network-level with these agda specs?